src/main/java/frc/robot/commands/CrocodileDefaultCommand.java

```diff
@@ -3,33 +3,49 @@
 import java.util.function.BooleanSupplier;
 import java.util.function.DoubleSupplier;
 
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import frc.robot.subsystems.CrocodileSubsystem;
 
 public class CrocodileDefaultCommand extends CommandBase {
     private final CrocodileSubsystem subsystem;
-    private final DoubleSupplier motorSpeed;
-    private final BooleanSupplier slowMovement;
+    private final DoubleSupplier trigger;
+    private final XboxController driverController;
+    private boolean beamBreakTracker = false;
+    private final Debouncer debouncer;
 
-    public CrocodileDefaultCommand(CrocodileSubsystem subsystem, DoubleSupplier motorSpeed,
-            BooleanSupplier slowMovement) {
+    public CrocodileDefaultCommand(CrocodileSubsystem subsystem, DoubleSupplier trigger) {
         this.subsystem = subsystem;
-        this.motorSpeed = motorSpeed;
-        this.slowMovement = slowMovement;
+        this.trigger = trigger;
+        //TODO: get driver controller port
+        //controller to use for rumble
+        driverController = new XboxController(0);
+        debouncer = new Debouncer(0.5);
```

Comment:
Not defining the debounce time willl have it default to `DebounceType.kRising`. Is that what you want? Check the modes section [here](https://docs.wpilib.org/en/stable/docs/software/advanced-controls/filters/debouncer.html).